helms/odahu-flow-core/templates/operator/decrypt_token.yaml

```diff
@@ -3,22 +3,22 @@
 apiVersion: v1
 kind: Secret
 metadata:
```

Comment:
We have the [Makefile](https://github.com/odahu/odahu-flow/blob/develop/packages/operator/Makefile#L77) that generates the CRDs. Please, fix it as well.

Reply:
done in 5a0e04d

Reply:
done in 5a0e04d